.circleci/config.yml

```diff
@@ -9,59 +9,45 @@ executors:
       - image: broadinstitute/womtool:84
         entrypoint: /bin/bash
     working_directory: ~/palantir-workflows
-    environment: WOMTOOL_JAR=/app/womtool.jar
 
 commands:
   get-cromwell-jar:
     description: "Download cromwell jar"
     steps:
       - run: wget https://github.com/broadinstitute/cromwell/releases/download/84/cromwell-84.jar
 
-  install-make:
-    description: "Install make"
+  install-watt:
+    description: "Install watt"
     steps:
       - run: |
-            apt-get update
-            apt-get install -y make
+            git clone -b ck_ignore_output https://github.com/kachulis/watt.git ~/watt
+            pip install -r ~/watt/requirements.txt
 jobs:
   validate:
     executor: womtool-executor
     steps:
       - checkout
-      - install-make
-      - run: make validate
+      - run: find . -name '*.wdl' -exec bash -c 'echo $1; java -jar /app/womtool.jar validate $1' shell {} \; 
 
-  validate-with-json:
-    executor: womtool-executor
-    steps:
-      - checkout
-      - install-make
-      - run: make validate-with-json
-
-  test:
+  test_watt:
     machine:
       image: ubuntu-2004:current
-    environment:
-      CROMWELL_JAR=/home/circleci/project/cromwell-84.jar
     steps:
       - checkout
       - get-cromwell-jar
-      - run: make test
-      - store_artifacts:
-          path: /home/circleci/project/logs
+      - install-watt
+      - run: |
+            python ~/watt/watt.py -e cromwell-84.jar -c test/watt_config.yml -p 2
       - store_artifacts:
-          path: /home/circleci/project/workflow_logs
+          path: watt_logs
       - store_artifacts:
-          path: /home/circleci/project/call_logs
-
+          path: cromwell-executions
```

Comment:
What effect does changing this have on finding/reading logs when tests fail in CI?

Reply:
should be somewhat easier, in that you have the entire cromwell execution directory now.  

that said, there isn't a ton of guidance when a test fails as to where to look for the failures (particularly if there is more than on test of the same workflow, figuring out which one to look in for the failure seems to be completely guess and check, atm).  I think that's something we could improve in watt.

Reply:
I was more confused about the change from a file path to a "local" path in the new commit. But yeah, for Watt it will tell you which tests failed and for what reason/outputs. Generally you shouldn't need the cromwell logs unless you have a failed test because the workflow failed to finish unexpectedly. So the Watt logs would generally be the place to look to see the test results summary, and then you can debug from there.